src/template.ejs

```diff
@@ -14,7 +14,7 @@
     <% } for (let js of htmlWebpackPlugin.files.js) { %>
     <script src="<%= path.relative(dir, js) %>" defer></script>
     <% } %>
-    <link href="<%= htmlWebpackPlugin.options.icon %>" rel="shortcut icon" type="image/png"/>
+    <link href="<%= htmlWebpackPlugin.options.icon %>" rel="shortcut icon" type="image/svg"/>
```

Comment:
Hmm, I don't like that the mimetype is hardcoded here, but that's my fault, so I'll just file an issue about it and fix it in a followup... :(